avocado/utils/software_manager.py

```diff
@@ -286,6 +286,26 @@ def list_files(self, name):
         except process.CmdError:
             return []
 
+    def rpm_install(self, file_path):
+        """
+        Install the rpm file [file_path] provided.
+
+        :param file_path: Rpm file path.
+        :return True: if file is installed properly
+        """
+        if os.path.isfile(file_path):
+            option = '-i'
+            cmd = 'rpm %s %s' % (option, file_path)
```

Comment:
simple `cmd = 'rpm -i %s' % file_path` would do here... (no need to overengeneer this)